### PR TITLE
Add support for token.scm_token in POST /person/<login>/token

### PR DESCRIPTION
**Pull request by dmarcoux**
The commit was cherry-picked from #11060 in order to merge changes which can already go on `master`.

### CODE REVIEW

File: src/api/public/apidocs-new/paths/person_login_token.yaml [hunk 1]
```diff
@@ -82,6 +82,15 @@ post:
 
         When operation is not specified, 'runservice' is the default value.
       example: runservice
+    - in: query
+      name: scm_token
+      schema:
+        type: string
+      description: |
+        SCM token used in OBS workflows, when the operation is workflow.
```
Comment (krauselukas) [inline src/api/public/apidocs-new/paths/person_login_token.yaml:90]:
```suggestion
        SCM token used in OBS workflows, to report back the status, when the operation is workflow.
```

Comment (krauselukas) [reply src/api/public/apidocs-new/paths/person_login_token.yaml:90]: Something like that, just to make the purpose of the scm token clear

File: docs/api/api/api.txt [hunk 2]
```diff
@@ -168,6 +168,7 @@ POST /person/<userid>/token
   project: together with package, used to create a token for a specific package
   package: together with project, used to create a token for a specific package
   operation: runservice(default), rebuild, release
+  scm_token: SCM token used in OBS workflows, when the operation is workflow
```
Comment (krauselukas) [inline docs/api/api/api.txt:171]:
```suggestion
  scm_token: SCM token used in OBS workflows, to report back the status, when the operation is workflow
```

**Comment by dmarcoux** [comment]
@krauselukas Please have another look. I've taken your suggestions into account and pushed changes.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**